tests/datastructures/test_network.py

```diff
@@ -7,3 +32,12 @@ def test_add_vertex():
     assert network.add_vertex(x=0, y=0, z=0) == 1
     assert network.add_vertex(key=2) == 2
     assert network.add_vertex(key=0, x=1) == 0
+
+
+def test_non_planar(k5_network):
```

Comment:
@brgcode at first, I used the test in `__main__` of `_planarity` for the unit test, but weirdly enough, that graph returns true for planarity on Python 2.7 and false on 3.x! So I reverted to a canonical planarity test case.